lib/driver/sdr_driver.c

```diff
@@ -3,6 +3,9 @@
 #include "fec.h"
 #include "osal.h"
 #include <stdio.h>
+#include "rfModeWrapper.h"
+#include "csp_types.h"
```

Comment:
I don't like that we included CSP in this repo.. I wonder if there's a better place to put it